packages/sanity/src/tasks/__telemetry__/tasks.telemetry.ts

```diff
@@ -0,0 +1,60 @@
+import {defineEvent} from '@sanity/telemetry'
+
+// Task is created.
+export const TaskCreated = defineEvent({
+  name: 'Task Created',
+  version: 1,
+  description: 'A task is created',
+})
+
+// Task status changed. %from% - %to%
+export const TaskStatusChanged = defineEvent<{
+  from: string
+  to: string
+}>({
+  name: 'Task Status Changed',
+  version: 1,
+  description: 'Task status changed',
+})
+
+// A comment is added in a task
+export const TaskCommentAdded = defineEvent({
+  name: 'Task comment added',
+  version: 1,
+  description: 'A comment was added in a task',
+})
+
+// A reply is added
+export const TaskCommentReplyAdded = defineEvent({
+  name: 'Task comment reply added',
```

Comment:
Suggestion: Add an "is_reply" boolean property to "Task Comment Added" to avoid two different events